frontend/src/components/UserProfile.vue

```diff
@@ -99,6 +99,15 @@ async function fetchData() {
   }
 }
 
+async function saveLanguage(locale: Locale) {
+  const updatedUser = me.value;
+  if (updatedUser !== undefined) {
+    updatedUser.language = locale.toString();
```

Comment:
is `locale.toString()` some form of standard? we might be stricter, only allowing iso language codes, so we can rely on the data not having inconsistencies due to strange browsers etc

Reply:
nevermind, just saw that Locale is our own enum. We should add an inline comment though to restrict the type (and can then also limit the varchar in the db)

Reply:
Fixed it [e0a09e9](https://github.com/cryptomator/hub/pull/313/commits/e0a09e94da41fdced50a997e7756cf3e80890660)